src/components/file/dto/node.ts

```diff
@@ -107,7 +107,7 @@ export class FileVersion extends BaseFile {
   static readonly Props = keysOf<FileVersion>();
   static readonly SecuredProps = keysOf<SecuredProps<FileVersion>>();
 
-  declare readonly type: FileNodeType.FileVersion;
+  declare readonly type: typeof FileNodeType.FileVersion;
```

Comment:
This is a use case I hadn't thought of. I think I'd rather just use the string literal in this case. It's less syntax and just as type safe.
```suggestion
  declare readonly type: 'FileVersion';
```
Same applies everywhere else too.